webapp/pom.xml

```diff
@@ -140,6 +140,12 @@
             <artifactId>micrometer-registry-statsd</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.springdoc</groupId>
+            <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
+            <version>2.6.0</version>
```

Comment:
Extract the version value into a maven property and then reference that property here, makes it straightforward for version updates in future.